os/net/netmgr/bsd_socket_api.c

```diff
@@ -23,21 +23,25 @@
  */
 
 #include <tinyara/config.h>
-#include <tinyara/cancelpt.h>
 
 #ifdef CONFIG_NET
-
+#include <tinyara/cancelpt.h>
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <netinet/in.h>
 #include <net/if.h>
 #include <tinyara/lwnl/lwnl.h>
 #include "netstack.h"
+#include "uds.h"
 
-/*  Private */
+#define CONFIG_NUDS_DESCRIPTORS 0 // to be fixed: temporary
+#define AF_UNIX 1000 // to be fixed: temporary
```

Comment:
This line can be removed if #4126 is merged.
How about reviewing #4126 first?

Reply:
ok

Reply:
Now this line can be removed.